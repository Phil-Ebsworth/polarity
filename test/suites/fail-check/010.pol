data Bool { T, F }

data Foo(a: Bool) {
    Bar: Foo(T),
    Baz: Foo(F)
}

def Foo(?).foo: Foo(F) {
    Bar => ?,
    Baz => ?
}
