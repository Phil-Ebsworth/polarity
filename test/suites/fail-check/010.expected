Cannot automatically decide whether T and ? unify